example/spring-batch/src/main/kotlin/com/linecorp/kotlinjdsl/example/spring/batch/entity/author/Author.kt

```diff
@@ -0,0 +1,21 @@
+package com.linecorp.kotlinjdsl.example.spring.batch.entity.author
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.Table
+import java.util.*
+
+@Entity
+@Table(name = "author")
+class Author(
+    @Id
+    @Column(name = "author_id")
+    val authorId: Long,
+
+    @Column(name = "name")
+    var name: String,
+) {
+    override fun equals(other: Any?): Boolean = Objects.equals(authorId, (other as? Author)?.authorId)
+    override fun hashCode(): Int = Objects.hashCode(authorId)
+}
```

Comment:
Please update the develop/3.0.0 branch and implement `toString()` on each Entity by referencing the `toString()` method in the other examples.

Reply:
I've added `toString()` method in Entities.